webhook_server/libs/handlers/runner_handler.py

```
@@ -0,0 +1,854 @@
+        finally:
+            yield result
+            self.logger.debug(f"{self.log_prefix} Deleting {clone_repo_dir}")
+            shutil.rmtree(clone_repo_dir, ignore_errors=True)
+
+    def is_podman_bug(self, err: str) -> bool:
+        _err = "Error: current system boot ID differs from cached boot ID; an unhandled reboot has occurred"
+        return _err in err.strip()
+
+    def fix_podman_bug(self) -> None:
+        self.logger.debug(f"{self.log_prefix} Fixing podman bug")
+        shutil.rmtree("/tmp/storage-run-1000/containers", ignore_errors=True)
```

Ruff (0.14.1)

Probable insecure usage of temporary file or directory: "/tmp/storage-run-1000/containers"

(S108)